packages/components/src/FieldMessage/_docs/FieldMessage.mdx

```diff
@@ -0,0 +1,68 @@
+import { Canvas, Controls, DocsStory, Meta } from "@storybook/blocks"
+import { ResourceLinks, KaioNotification, Installation } from "~storybook/components"
+import * as FieldMessageStories from "./FieldMessage.stories"
+
+<Meta of={FieldMessageStories} />
+
+# FieldMessage
+
+{/*
+  * @todo:
+  * - Adjust Github link (if needed)
+  * - Add Figma and Design Guidelines (Confluence) link
+  */}
```

Comment:
Remove

Reply:
Ah yep, nice catch, will axe the boilerplate comments 😉